src/extras/core/Interpolations.d.ts

```diff
@@ -0,0 +1,5 @@
+export namespace Interpolations {
+	export function CatmullRom(): any;
+	export function QuadraticBezier(): any;
+	export function CubicBezier(): any;
```

Comment:
Using `any` is bad form for typescript definitions. These functions all return a `Number` type.

Reply:
Update: I mistyped -- it looks like it should be `number`.

https://www.typescriptlang.org/docs/handbook/basic-types.html

Reply:
OK, I change it immediately.